<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <node name="urc_lane_follower" pkg="rr_iarrc" type="urc_lane_follower" output="screen">
        <param name="speed" type="double" value="1.0"/>
        <param name="PID_kP" type="double" value="-0.01"/>
        <param name="left_angle_offset" type="double" value="1.0"/>
        <param name="right_angle_offset" type="double" value="-1.0"/>
        <param name="camera_left_subscription" type="string" value="/camera_left/image_color_rect"/>
        <param name="camera_right_subscription" type="string" value="/camera_right/image_color_rect"/>
    </node>
</launch>
